bin/fin

```diff
@@ -4643,7 +4643,7 @@ ngrok_share ()
 		--link ${container_name} \
 		--name ${ngrok_container_name} \
 		wernight/ngrok \
-		"$ARGS"
+		$ARGS
```

Comment:
Why was this changed?

Reply:
@achekulaev The `$ARGS` is wrapped in quotes which is causing issues with duplicate quoting. The args are meant to be sent separately. Do you think that with or without the quotes it shouldn't matter? Also the arguments have spaces between them when need to be sent as arguments to the `ngrok` command.

Reply:
Ah right. Quoting them signals bash it's a single value. OK. Thanks